app/views.py

```diff
@@ -0,0 +1,231 @@
+from django.http import HttpResponse
+from django.shortcuts import redirect
+from .models import Board, Card, BoardList
+from django.contrib.auth.models import User
+from django.http import HttpResponseRedirect
+from django.views.generic import TemplateView
+from .forms import BoardForm, ListForm, CardForm, UserForm
+from django.contrib.auth import authenticate, login
+from django.shortcuts import render, get_object_or_404
+
+
+# Create user view class
+class LoginView(TemplateView):
+    """
+    Let user to login
+    """
+
+    template_name = 'app/login.html'
+    form = UserForm
+    def get(self, *args, **kwargs):
+        form = self.form()
+        return render(self.request, self.template_name,{ 'form' : form})
+
+    def post(self, *args, **kwargs):
+        username = self.request.POST['username']
+        password = self.request.POST['password']
+        user = authenticate(self.request, username=username, password=password)
+        if user is not None:
+            login(self.request, user)
+            return HttpResponseRedirect('/')
+        #  else:
```

Comment:
what happens if user is equal to None? make sure to handle errors

Reply:
Done sir